targetcli/ui_backstore.py

```diff
@@ -651,6 +651,12 @@ def ui_command_create(self, name, size, cfgstring, wwn=None,
                             % (name, size))
         return self.new_node(ui_so)
 
+    def ui_command_changemedium(self, name, size, cfgstring):
```

Comment:
Were you going to move this to on the device so you do not have to pass in the name?

> 
> /backstores/user:file> ls
> o- user:file .............................................. [Storage Objects: 2]
>   o- lun0 ................................. [/work/luns/lun0 (1.0GiB) activated]

So instead of the backstore "user:file" it seems like it should be on the storage object lun0.

Also, do we need a try/catch somewhere? Maybe around the self.iface.ChangeMedium call. With the patch with the original naming, I killed runner while executing the command and I get this crash:

> /backstores/user:file> mediachange lun1 1G /work/luns/lun2
> Traceback (most recent call last):
>   File "/usr/bin/targetcli", line 5, in <module>
>     pkg_resources.run_script('targetcli-fb==2.1.fb48', 'targetcli')
>   File "/usr/lib/python2.7/site-packages/pkg_resources.py", line 540, in run_script
>     self.require(requires)[0].run_script(script_name, ns)
>   File "/usr/lib/python2.7/site-packages/pkg_resources.py", line 1462, in run_script
>     exec_(script_code, namespace, namespace)
>   File "/usr/lib/python2.7/site-packages/pkg_resources.py", line 41, in exec_
>     exec("""exec code in globs, locs""")
>   File "<string>", line 1, in <module>
>   File "/usr/lib/python2.7/site-packages/targetcli_fb-2.1.fb48-py2.7.egg/EGG-INFO/scripts/targetcli", line 122, in <module>
>     
>   File "/usr/lib/python2.7/site-packages/targetcli_fb-2.1.fb48-py2.7.egg/EGG-INFO/scripts/targetcli", line 112, in main
>     
>   File "/usr/lib/python2.7/site-packages/configshell_fb-1.1.24-py2.7.egg/configshell_fb/shell.py", line 905, in run_interactive
>     self._cli_loop()
>   File "/usr/lib/python2.7/site-packages/configshell_fb-1.1.24-py2.7.egg/configshell_fb/shell.py", line 734, in _cli_loop
>     self.run_cmdline(cmdline)
>   File "/usr/lib/python2.7/site-packages/configshell_fb-1.1.24-py2.7.egg/configshell_fb/shell.py", line 848, in run_cmdline
>     self._execute_command(path, command, pparams, kparams)
>   File "/usr/lib/python2.7/site-packages/configshell_fb-1.1.24-py2.7.egg/configshell_fb/shell.py", line 823, in _execute_command
>     result = target.execute_command(command, pparams, kparams)
>   File "/usr/lib/python2.7/site-packages/configshell_fb-1.1.24-py2.7.egg/configshell_fb/node.py", line 1406, in execute_command
>     return method(*pparams, **kparams)
>   File "build/bdist.linux-x86_64/egg/targetcli/ui_backstore.py", line 663, in ui_command_mediachange
>   File "/usr/lib64/python2.7/site-packages/gi/overrides/Gio.py", line 171, in __call__
>     None)
> GLib.Error: g-dbus-error-quark: GDBus.Error:org.freedesktop.DBus.Error.NoReply: Message did not receive a reply (timeout by message bus) (4)